dlk/tests/onnx/test_onnx_io.py

```diff
@@ -1,108 +0,0 @@
-# -*- coding: utf-8 -*-
-# Copyright 2018 The Blueoil Authors. All Rights Reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-# =============================================================================
-"""Test file for OnnxIO."""
-from os import path
-import unittest
-
-from core.graph import Graph
-
-
-def onnx_is_available() -> bool:
-    available = True
-    try:
-        __import__('onnx')
-    except ImportError:
-        available = False
-
-    return available
-
-
-class TestOnnxIO(unittest.TestCase):
-    """Test class for OnnxIO."""
-
-    @unittest.skipUnless(onnx_is_available(), "ONNX is not available (reinstall with --enable-onnx)")
-    def test_onnx_import(self) -> None:
-        """Test code for importing ONNX file with OnnxIO."""
-        from frontend.onnx import OnnxIO
-
-        onnx_path = path.join('examples',
-                              'classification',
-                              'lmnet_quantize_cifar10_stride_2.20180523.3x3')
-
-    @unittest.skipUnless(onnx_is_available(), "ONNX is not available (reinstall with --enable-onnx)")
-    def future_test_onnx_import_lmnet_classification(self) -> None:
-        """Test code for importing lmnet classification via ONNX."""
-        # Current lmnet classification onnx has several problems,
-        # so we leave this test for the near future.
-        from frontend.onnx import OnnxIO
-
-        onnx_path = path.join('examples',
-                              'classification',
-                              'lmnet_quantize_cifar10_stride_2.20180523.3x3',
-                              # 'lmnet_quantize_cifar10_max_pooling',
-                              'onnx_minimal_graph_with_shape.pb')
```

Comment:
@iizukak Can we also remove the test `.pb` file?

Reply:
@nlpng Oh, we should remove. thanks.

Reply:
@nlpng Removed at fd306bba60ba1ac9f1f4f4c2e776f44def2efedd